include/opendht/dhtrunner.h

```diff
@@ -362,6 +366,18 @@ class OPENDHT_PUBLIC DhtRunner {
      */
     void join();
 
+#if OPENDHT_PROXY_SERVER
+    /**
+     * Start the proxy server
+     * @param port used for the server
+     */
+    void startProxyServer(unsigned int port=8000);
```

Comment:
Maybe the proxy server could be a external API ?
Only the proxy client need to change the DhtRunner API (startProxy/stopProxy).

This would allow to run multiple proxies on a single DHT node.
```c++
auto node = make_shared<DhtRunner>();
// start the servers
auto proxy_server1 = make_unique<DhtProxyServer>(node, 80);
auto proxy_server2 = make_unique<DhtProxyServer>(node, 8080);
// stop the server 1
proxy_server1.reset();
```

Reply:
Indeed that's much better

Reply:
Done